dora/tests/src/test/java/alluxio/client/cli/fs/AbstractShellIntegrationTest.java

```diff
@@ -30,7 +30,7 @@
  * The base class for all the shell integration test.
  */
 public abstract class AbstractShellIntegrationTest extends BaseIntegrationTest {
-  protected static final int SIZE_BYTES = Constants.MB * 16;
+  protected static final int SIZE_BYTES = Constants.MB * 1024;
```

Comment:
relevant?

Reply:
I found that some tests have to write large files that are larger than 16MB. This makes them failed. So I just set it larger to avoid failure.

Reply:
we should fix the failure of large files instead of making the failure go away

Reply:
I think instead of targeting to fix those tests in this change, we should merge this change in ASAP to unblock other changes. And track the disabled tests and strive to add them back with priority.

Reply:
@JiamingMai could you run again with `SIZE_BYTES = Constants.MB * 16` and paste the error here, so we know what we are hacking